portal/academy/views.py

```diff
@@ -160,12 +160,16 @@ class InstructorUserListView(InstructorViewsMixin, ListView):
     def get_queryset(self):
         user_id = self.request.GET.get("user_id")
         can_graduate = self.request.GET.get("can_graduate")
+        can_attend_next = self.request.GET.get("can_attend_next")
 
         if can_graduate is not None:
             return self.queryset.filter(can_graduate=can_graduate).order_by(
                 "name"
             )
 
+        if can_attend_next is not None:
+            return self.queryset.filter(can_graduate=can_attend_next).order_by("name")
```

Comment:
This should filter `can_attend_next`.

Also if both `can_graduate` and `can_attend_next` are selected only `can_graduate` is applied, is this by design? Only students that `can_graduate` `can_attend_next`?

Reply:
> Only students that can_graduate can_attend_next? 


This makes sense to me, but exceptions might happen. is this something we can overwrite in the admin portal?